schemas/fingerprint-server-api.yaml

```diff
@@ -714,6 +760,24 @@ components:
                   description: >
                     `true` if the request IP address is a known tor exit node, `false` otherwise.
                   example: false
+            privacySettings:
+              title: WebhookSignalResponsePrivacySettings
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    `true` if the request is from privacy aware browser (e.g. Brave with fingerprinting blocking), `false` otherwise.
+                  example: false
+            virtualMachine:
+              title: WebhookSignalResponseVirtualMachine
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    `true` if the request is under virtual machine (e.g. VMWare), `false` otherwise.
```

Comment:
```suggestion
                    `true` if the request came from a browser running inside a virtual machine (e.g., VMWare), `false` otherwise.
```

Would it be accurate to rephrase it like this?

Reply:
done